pandas/io/stata.py

```diff
@@ -1897,7 +1930,7 @@ def _convert_datetime_to_stata_type(fmt):
         raise NotImplementedError(f"Format {fmt} not implemented")
 
 
-def _maybe_convert_to_int_keys(convert_dates, varlist):
+def _maybe_convert_to_int_keys(convert_dates: Dict, varlist: List[Label]) -> Dict:
```

Comment:
If possible to add subtypes to Dict would be very helpful

Reply:
Thanks, added.